src/core/frontend/manager/UIManager.cpp

```diff
@@ -31,7 +32,12 @@ namespace YimMenu
 		}
 		ImGui::EndChild();
 		
-		ImGui::Text("HorseMenu");
+		std::string gameVer = DEBUG::GET_GAME_VERSION_NAME();
```

Comment:
Are you sure this works? Calling natives in the rendering thread doesn't usually work.

Reply:
1000% sure it works as expected unless the game changed since adding this.  It shows the game version at the bottom left of the gui.

Reply:
> Are you sure this works? Calling natives in the rendering thread doesn't usually work.

Natives which just return class members (i.e GET_PLAYER_NAME just returns CName) can be called on the DX thread as those don't require TLS. It's not good practice, but can be done. Other natives which need a script thread TLS will still crash when called on the DX thread.

I think GET_GAME_VERSION just prints whatever value is in the update.rpf gameversion.txt if I remember right so won't need a script's TLS given it's a file read operation.